compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/ComposeContainer.desktop.kt

```diff
@@ -78,6 +79,7 @@ import org.jetbrains.skiko.SkiaLayerAnalytics
  *  for window coordinate space.
  * @property useSwingGraphics Flag indicating if offscreen rendering to Swing graphics is used.
  * @property layerType The type of layer used for Popup/Dialog.
+ * @property renderingSettings The settings for rendering.
```

Comment:
Add a note that it will work only with `useSwingGraphics == true`

Reply:
It's in the doc RenderSettings.

Reply:
In general it's not true, because we can add settings, that _will_ work with swing